test/facets/third-party-facet.spec.ts

```diff
@@ -50,24 +50,23 @@ describe("Contract: ThirdPartyFacet", () => {
         // after cleaning, calculate sum, and verify on exact amounts instead
         // Set up amounts that should have been transferred to accuser and embalmer
         // ie, rewards of failed archaeologists
-        const totalDiggingFeesAndBounty = archaeologists
+        const sumDiggingFees = archaeologists
           .slice(1) // archaeologists[0] did their job, so not included
           .reduce(
-            (acc, arch) => [acc[0].add(arch.diggingFee), acc[1].add(arch.bounty)],
-            [BigNumber.from("0"), BigNumber.from("0")]
+            (acc, arch) => [acc[0].add(arch.diggingFee)],
+            [BigNumber.from("0")]
           );
 
-        const totalDiggingFees = totalDiggingFeesAndBounty[0];
-        const totalBounty = totalDiggingFeesAndBounty[1];
+        const totalDiggingFees = sumDiggingFees[0];
```

Comment:
We may not need both these vars now (`sumDiggingFees` / `totalDiggingFees`) -- will look into combining in a followup PR.